contracts/callpaths/ColdPath.sol

```diff
@@ -172,15 +172,18 @@ contract ColdPath is MarketSequencer, DepositDesk, ProtocolAccount {
      * @param tickSize The pool's grid size in ticks.
      * @param jitThresh The minimum resting time (in seconds) for concentrated LPs.
      * @param knockout The knockout bits for the pool template.
-     @ @param oracleFlags The oracle bit flags if a permissioned pool. */
+     * @param oracleFlags The oracle bit flags if a permissioned pool. 
+     * @param bidTick The tick index of the stable swap pool's bid price.
+     * @param askTick The tick index of the stable swap pool's ask price.
```

Comment:
Seeing as how its a limited range, maybe bidTickFloor, and askTickCeiling might make more sense